app/views/idv/gpo/index.html.erb

```diff
@@ -9,7 +9,9 @@
       ) %>
 <% end %>
 
-<%= render PageHeadingComponent.new.with_content(@presenter.title) %>
+<%= render AlertComponent.new(message: @presenter.info_alert_message, class: 'margin-bottom-4') %>
+
+<%= render PageHeadingComponent.new(class: 'margin-bottom-4').with_content(@presenter.title) %>
```

Comment:
One of the primary objectives of the `PageHeadingComponent` component is to standardize expected margins for headings. Do we need the ad hoc margin, or can we let it inherit the default standard margins that we use throughout the application?

Reply:
Looking at the [Figma](https://www.figma.com/file/pfd0qWEfLjuEhZ0p1oZDa5/IDV-Vendor-Outages---Error-Messaging?node-id=501%3A26043&t=1hvCM3YNCdCIyKDJ-0), we need the 4 unit margin, either on the bottom of the heading or on the top of the paragraph (I used the new alt trick, thanks)

Reply:
I'm wondering if there might be some confusion here caused by how forms are [configured with default top margin](https://github.com/18F/identity-idp/blob/203cab85de5137082e115970403138134757d366/config/initializers/simple_form.rb#L9) that's slightly larger than the [heading's default bottom margin](https://github.com/18F/identity-idp/blob/203cab85de5137082e115970403138134757d366/app/assets/stylesheets/components/_page-heading.scss#L3), so some pages may have more or less margin between the heading and content. I'd actually [planned to raise this for discussion](https://docs.google.com/document/d/10b7Gis_VxYT3roR01MX4hwPa6-Tq29YBaLPIB3vWnUw/edit#bookmark=id.ac2mjjnowbhd) as a Design System sync topic, since I think it contributes to unclarity of standard margins.

In the meantime, I'd personally be inclined to bring this back to UX to confirm that there's something special about this screen that we are intentionally diverging from the standard margin, since my sense is that we shouldn't.